README.md

```diff
@@ -21,14 +21,45 @@ Once you have implemented the `fm` program (and tested it to the best of your ab
 
 *What preprocessing data do you store in files, and how?*
 
+For pre-processing we ended up using pickle, there was some amount of fidling around with storing stuff in simple txt files. But hey, why invent a way to pack and parse stuff when those already exist. 
```

Comment:
It is indeed unnecessary to reinvent the wheel, especially when your invention easily end up being square.